smaug/python/ops/common.py

```diff
@@ -3,6 +3,7 @@
 from smaug.core import types_pb2
 from smaug.python import global_vars
 from smaug.python import tensor_utils
+from smaug.python.ops import array_ops
```

Comment:
I thought `array_ops.py` would include `common.py`, so this will create a cycle dependency?

Reply:
Oh yeah. I moved the other two functions to array_ops instead, so now common.py only has this one add_node function.